packages/maskbook/src/plugins/NFT/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface NFTInPostProps {
+    nftUrl: string
+}
+
+export interface TokenDetails {
+    name: string | null
```

Comment:
why not optional property?

see https://www.typescriptlang.org/docs/handbook/interfaces.html#optional-properties

Reply:
see https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining